src/linker/Linker/LinkerFatalErrorException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Mono.Linker
+{
+	public class LinkerFatalErrorException : Exception
```

Comment:
Maybe it'd be good to add some comments about what this is for, the intended use, and how it's different from InternalErrorException - and same for InternalErrorException.